### PR TITLE
[MP-614] ChatSDK v0.2.0

### PR DESCRIPTION
**Pull request by SamborGornicz**
# Links

- [Jira](https://livechatinc.atlassian.net/browse/MP-614)

# Description

Align ChatSDK documentation with the newest changes.

### CODE REVIEW

**Review by ale-kacperczyk — COMMENTED** [review]
Can we also add this link: https://developers.livechat.com/docs/authorization/authorizing-api-calls#personal-access-tokens
as a bullet on this list?
image

File: src/pages/messaging/chat-sdk/index.mdx [hunk 1]
```diff
@@ -188,21 +194,23 @@ chatSDK.sendMessage(chatId, message)
 ## Creating custom methods
 This SDK supports the RTM transport. For that reason, make sure you use the [Agent Chat API RTM](/messaging/agent-chat-api/rtm-reference/) reference. 
 When creating your custom methods, base on the payloads from the [Agent Chat RTM API methods](/messaging/agent-chat-api/rtm-reference/#methods).
-
-In the example below, we're creating a custom method that returns chat thread summaries.
-As you can see in the [documentation](/messaging/agent-chat-api/rtm-reference/#get-chat-threads-summary), only `chat_id` is required, but you can include other optional parameters in your custom method.
+​
+In the example below, we're creating a custom method that returns chat thread summaries.
```
Comment (ale-kacperczyk) [inline src/pages/messaging/chat-sdk/index.mdx:198]:
```suggestion
In the example below, we're creating a custom method that returns a chat.
```

**Review by ale-kacperczyk — COMMENTED** [review]
One final comment :)

**Review by ale-kacperczyk — APPROVED** [review]

**Comment by ale-kacperczyk** [comment]
Let me know on #platform-docs-releases when this PR is ready for merging

**Review by arturfracala — APPROVED** [review]

**Pull request merged**